docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -41,7 +41,11 @@ dnf --assumeyes install make gcc gcc-c++ glibc-devel git-core libsecret-devel rp
 #### Arch
 
 ```sh
-TODO: Arch instructions
+# Make sure you update the system
+pacman -Syu
```

Comment:
We don't have this in place for the others so I don't think this should be necessary for dependency install instructions. I'm not sure we want the responsibility to tell somebody to upgrade their entire system.

Reply:
Will fix

Reply:
I fixed them.